unlock-app/next.config.js

```diff
@@ -71,6 +73,10 @@ module.exports = withTypescript({
       '/terms': { page: '/terms' },
       '/privacy': { page: '/privacy' },
       '/log': { page: '/log' },
+      '/post': { page: '/post' },
```

Comment:
I recommend making `Blog` a kind of React-based router. It would accept a slug as a param, and either render the blog index or a post based on its presence (similar to what I ended up doing for the paywall landing page). This avoids the need to monkey with next pages, since users will never explicitly navigate to `/post/whatever`